code/messaging/src/main/java/com/adobe/marketing/mobile/MessagingPushBuilder.java

```diff
@@ -129,10 +130,10 @@ private static void addActionButtons(final NotificationCompat.Builder builder,
      * @param builder the notification builder
      */
     private static void setSmallIcon(final NotificationCompat.Builder builder,
-                              final MessagingPushPayload payload,
-                              final Context context) {
+                                     final MessagingPushPayload payload,
+                                     final Context context) {
         final int iconFromPayload = MessagingPushUtils.getSmallIconWithResourceName(payload.getIcon(), context);
-        final int iconFromMobileCore =  -1 ;   //MobileCore.getSmallIconResourceId();
```

Comment:
maybe we can define a constant for `-1` so it's use is more apparent

Reply:
I will have to -1 with `MobileCore.getSmallIconResourceId()` API. I will update them in the next PR when snapshot of MobileCore is available with the API